src/TetMesh.h

```diff
@@ -40,13 +40,28 @@ class TetMesh
         int m_timestamp = 0;
 
     public:
+        /**
+         * @brief Generate a Tuple from global tetra index and __local__ edge index (from 0-5).
```

Comment:
The `@brief` is not needed. You can have doxygen use the first sentence as the brief description.